stdlib/runtime/wasi.gr

```diff
@@ -217,120 +218,219 @@ provide foreign wasm path_rename: (
 ) => WasmI32 from "wasi_snapshot_preview1"
 
 // clocks
+@unsafe
```

Comment:
Thoughts on allowing an entire module to be marked as unsafe?

Reply:
Hmm, I'm not a big fan of that personally. I think requiring that it be explicit helps add context that the thing you're doing is unsafe, and I think we generally do not want to encourage users to use unsafe at all which allowing this attribute at the module level would make easier

Reply:
Fair enough